src/main/java/com/mongodb/jdbc/MongoListCollectionsResult.java

```diff
@@ -0,0 +1,6 @@
+package com.mongodb.jdbc;
+
+public class MongoListCollectionsResult {
```

Comment:
There are other fields returned by this, but these are the only relevant ones for this PR. We can extend this class as needed in the future.

Reply:
sgtm, I assume you tested this, but at the same time, it's so simple it shouldn't be a problem.

Reply:
yup, tested it!